pkgs/top-level/all-packages.nix

```diff
@@ -6361,6 +6361,8 @@ in
 
   libscrypt = callPackage ../development/libraries/libscrypt { };
 
+  libcsys = libsForQt5.callPackage ../development/libraries/libcsys { };
```

Comment:
You are missing libcprime.

Reply:
> You are missing libcprime.

https://github.com/NixOS/nixpkgs/pull/130737

Reply:
libcprime There, I understand that you start these 2 PR at the same time and they will fall into the channel at one time. And everything will work